src/main/webapp/scheduled-interviews.js

```diff
@@ -12,6 +12,95 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 function onScheduledInterviewsLoad() {
   supplyLogoutLink();
+  listScheduledInterviews(); 
+}
+
+function listScheduledInterviews() {
+  var email = "gswe@gmail.com"; 
+  const scheduledInterviewsSection = document.getElementById('scheduled-interviews-cards');
+  document.getElementById('scheduled-interviews-cards').innerHTML = ''; 
+  fetch(`/scheduled-interviews?userEmail=${email}`)
+    .then(response => response.json())
+    .then((scheduledInterviews) => {
+      if (isEmptyObject(scheduledInterviews)){
+        const emptyEventsMessageDiv = document.createElement('div');
+        const  emptyEventsMessage = document.createElement('h2');
+        emptyEventsMessage.style.textAlign = 'center';  
+        emptyEventsMessage.innerHTML = "No Scheduled Interviews"; 
+        emptyEventsMessageDiv.appendChild(emptyEventsMessage); 
+        scheduledInterviewsSection.appendChild(emptyEventsMessage); 
+      } else { 
+        scheduledInterviews.forEach((scheduledInterview) => {
+          scheduledInterviewsSection.appendChild(createScheduledInterviewCard(scheduledInterview));
+          console.log(scheduledInterview); 
+        })
+      }
+    }); 
 }
+
+function createScheduledInterviewCard(scheduledInterview) {
+  // Check the role of the current user for the scheduledInterview
+  var email = "gswe@gmail.com";
+  var role; 
+  if(email === scheduledInterview.interviewerEmail) {
+    role = "Interviewer"; 
```

Comment:
These are strings, see if there is a .equals() in javascript.

Reply:
There is a .equals() method in JS but gives me a type error when I use it in this case: 

TypeError: email.equals is not a function at createScheduledInterviewCard at scheduled-interviews.js.38
at Array.forEach (<anonymous>)
at scheduled-interview.js.37